File: 1/list-1.go
package main
import "fmt"

type Carro struct {
    Marca   string
    Modelo  string
    Ano     int
}

func (c Carro) Detalhes() string {
    return fmt.Sprintf("Marca: %s, Modelo: %s, Ano: %d", c.Marca, c.Modelo, c.Ano)
}

func main() {
    carro1 := Carro{"Toyota", "Corolla", 2020}
    carro2 := Carro{"Honda", "Civic", 2018}
    carro3 := Carro{"Ford", "Focus", 2019}

    fmt.Println(carro1.Detalhes())
    fmt.Println(carro2.Detalhes())
    fmt.Println(carro3.Detalhes())
}

File: 1/list-1.py
class Carro:
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    def exibir_detalhes(self):
        print(f'Marca: {self.marca}, Modelo: {self.modelo}, Ano: {self.ano}')

def main():
    carro1 = Carro("Toyota", "Corolla", 2020)
    carro2 = Carro("Honda", "Civic", 2019)
    carro3 = Carro("Ford", "Mustang", 2022)

    carro1.exibir_detalhes()
    carro2.exibir_detalhes()
    carro3.exibir_detalhes()

if __name__ == "__main__":
    main()
